WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -37,6 +34,63 @@ struct ItemListView: View {
 /// View Helpers
 ///
 private extension ItemListView {
+    @ViewBuilder
```

Comment:
Thoughts on using `ViewBuilder` vs computed variables for the views? I tend to use the first one if there's some conditional logic or we need to pass params, but doesn't seem to be needed for all cases 🤔

Reply:
You can put @ViewBuilder for both functions and computed variables